src/main/java/fi/metatavu/dcfb/server/rest/LocationsApiImpl.java

```diff
@@ -0,0 +1,212 @@
+package fi.metatavu.dcfb.server.rest;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+import java.util.Arrays;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Arrays'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=11&issues=AWQt_bhXYwZU9vvrs3Kp&open=AWQt_bhXYwZU9vvrs3Kp)

[ISSUE_KEY]: AWQt_bhXYwZU9vvrs3Kp